docs/_includes/home-lhn.html

```diff
@@ -1,6 +1,6 @@
 <ul class="lhn-items">
     <li class="home-link">
-        <div class="link" onclick="navigateToHome()">Home</div>
+        <div class="selected">Home</div>
     </li>
     <li>
         <div onclick="navigateTo('/hubs/send-money')" class="icon-with-link">
```

Comment:
Looks like this isn't using an anchor either. Could you update it please (and any other clickable divs)?

Reply:
Ah yeah, missed this one. I updated all the remaining clickable divs in the code.